charts/seed-controlplane/charts/kube-apiserver/templates/deployment.yaml

```diff
@@ -194,7 +194,11 @@ spec:
               value: Bearer {{ .Values.probeCredentials }}
           successThreshold: 1
           failureThreshold: 3
+          {{- if semverCompare ">= 1.16" .Values.kubernetesVersion }}
+          initialDelaySeconds: 60
```

Comment:
I'm confused. How does changing the `initialDelaySeconds` of the liveness probe help in achieving the above goal (delaying the ready status)?
The liveness probe is not considered for the ready status, if I got it correctly. Also see https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/.

E.g., when applying this Pod:
```yaml
apiVersion: v1
kind: Pod
metadata:
  labels:
    test: liveness
  name: liveness-exec
spec:
  terminationGracePeriodSeconds: 1
  containers:
  - name: liveness
    image: k8s.gcr.io/busybox
    args:
    - /bin/sh
    - -c
    - sleep 30; touch /tmp/healthy; sleep 30; rm -rf /tmp/healthy; sleep 600
    livenessProbe:
      exec:
        command:
        - cat
        - /tmp/healthy
      initialDelaySeconds: 30
      periodSeconds: 5
    readinessProbe:
      exec:
        command:
        - echo
        - ready
      initialDelaySeconds: 5
      periodSeconds: 5
```
it gets ready after the 5 seconds delay for the readiness probe even though the liveness probe is delayed for 30s.